tests/integration/bond_test.py

```diff
@@ -171,112 +204,54 @@ def test_rollback_for_bond(eth1_up, eth2_up):
     assert current_state == current_state_after_apply
 
 
-def test_add_slave_to_bond_without_slaves(eth1_up, setup_remove_bond99):
-    bond_state = {
-            INTERFACES: [
-                {
-                    'name': 'bond99',
-                    'type': 'bond',
-                    'state': 'up',
-                    'link-aggregation': {
-                        'mode': 'balance-rr',
-                        'slaves': []
-                    },
-                }
-
-            ]
-        }
+def test_add_slave_to_bond_without_slaves(eth1_up):
 
-    netapplier.apply(bond_state)
+    with bond_interface('bond99', []) as bond_state:
```

Comment:
If I'm not mistaken, Till asked to use kwargs when calling the contextmanager:
`with bond_interface(name='bond99', slaves=[]) as bond_state:`